tools/c7n_azure/c7n_azure/function_package.py

```diff
@@ -48,9 +48,9 @@ def create_zinfo(self, file):
 
 
 class FunctionPackage(object):
+    log = logging.getLogger('custodian.azure.function_package')
```

Comment:
Should this be `FunctionPackage`?

Reply:
I only specified the class in files where other major classes were in the module and logging as well.  But for consistency I'll go through and just add the class name to everything.